### DISCUSSION TITLE
Mapping `<C-…>` vs `<c-…>`

### DISCUSSION THREAD

**Discussion by ernstwi** [LazyVim/LazyVim#1461] [Q&A]
Hi!

I noticed that in `keymaps.lua`, a mix of capital and lowercase C is used in the `lhs` of mappings. Wondering if there's a reason for this, as far as I know they do the same thing?

**Comment by abeldekat** [comment]
Hi @ernstwi,

I don't think there is a reason. When inspecting the resulting `keymap`, via `Telescope keymaps` for example, the lowercase `c` is also shown in uppercase.
